network.js

```diff
@@ -2706,6 +2706,20 @@ function handleRequest(ws, tag, command, params){
 			});
 			break;
 
+		case 'light/get_definition':
+			if (conf.bLight)
+				return sendErrorResponse(ws, tag, "I'm light myself, can't serve you");
+			if (ws.bOutbound)
+				return sendErrorResponse(ws, tag, "light clients have to be inbound");
+			if (!params)
+				return sendErrorResponse(ws, tag, "no params in light/get_definition");
+			if (!ValidationUtils.isValidAddress(params))
+				return sendErrorResponse(ws, tag, "address not valid");
+			db.query("SELECT * FROM definitions WHERE definition_chash=?", [params], function(rows){
```

Comment:
better select only the definition, otherwise the response is more than a definition